LIBSRC/OSS/oss_sem.c

```diff
@@ -237,7 +237,7 @@ int32 OSS_SemWait(
 
 	/* sem->lock is locked here */
 	{
-		wait_queue_t __wait;
+		wait_queue_entry_t __wait;
```

Comment:
not present in the kernel 3.6.11

Reply:
I'll let the change apply for kernel version bigger than 4.13.